md_api/src/main/java/com/wipro/ats/bdre/md/api/InitStep.java

```diff
@@ -46,8 +47,6 @@ public InitStep() {
      * @param params String array having environment and sub-process-id with their command line notations
      * @return returns sub-process-instance-exec-id.
      */
-    @Autowired
-    private StepDAO stepDAO;
 
     public InitStepInfo execute(String[] params) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1161)